protos/follow_me/follow_me.proto

```diff
@@ -30,19 +30,23 @@ service FollowMeService {
 
 // Configuration type.
 message Config {
-    // Direction relative to the target that the vehicle should follow
-    enum FollowDirection {
-        FOLLOW_DIRECTION_NONE = 0; // Do not follow
-        FOLLOW_DIRECTION_BEHIND = 1; // Follow from behind
-        FOLLOW_DIRECTION_FRONT = 2; // Follow from front
-        FOLLOW_DIRECTION_FRONT_RIGHT = 3; // Follow from front right
-        FOLLOW_DIRECTION_FRONT_LEFT = 4; // Follow from front left
+    // Follow Perspective that the vehicle should follow
+    enum FollowPerspective {
+        FOLLOW_PERSPECTIVE_BEHIND = 0;      // Follow from behind
+        FOLLOW_PERSPECTIVE_FRONT = 1;       // Follow from front
+        FOLLOW_PERSPECTIVE_FRONTRIGHT = 2;  // Follow from front right
+        FOLLOW_PERSPECTIVE_FRONTLEFT = 3;   // Follow from front left
+        FOLLOW_PERSPECTIVE_RIGHT = 4;       // Follow from right
+        FOLLOW_PERSPECTIVE_LEFT = 5;        // Follow from left
+        FOLLOW_PERSPECTIVE_BEHINDRIGHT = 6; // Follow from behind right
+        FOLLOW_PERSPECTIVE_BEHINDLEFT = 7 ; // Follow from behind left
+        FOLLOW_PERSPECTIVE_INVALID = 8;     // Follow perspective invalid
```

Comment:
Adding fields is fine, changing them is a bit more tricky, and - as @JonasVautherin said - probably not worth the trouble.

Reply:
Gotcha, wasn't aware of that! Protobuf stuff :mask: